libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/Actions/Microsoft.SwitchCondition.schema

```diff
@@ -11,30 +11,34 @@
         "condition": {
             "$role": "expression",
             "title": "Condition",
-            "description": "Expression to evaluate to switch on.",
+            "description": "Expression to evaluate.",
             "examples": [
-                "user.age > 3"
+                "user.favColor"
             ]
         },
         "cases": {
             "type": "array",
             "title": "Cases",
-            "desc": "Cases to evaluate against condition",
+            "desc": "Actions for each possible condition.",
             "items": {
                 "type": "object",
                 "properties": {
                     "value": {
                         "$role": "expression",
                         "title": "Value",
-                        "description": "Value which must match the condition property"
+                        "description": "Condition value.",
```

Comment:
>Condition value [](start = 40, length = 15)

What is wrong with simply Value?

Reply:
updated. We could use this to re-iterate (through the DCR) that this is actually a possible value of the property.